srcpkgs/luksmeta/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'luksmeta'
+pkgname=luksmeta
+version=9
+revision=1
+build_style=gnu-configure
+hostmakedepends="asciidoc automake libtool pkg-config"
+makedepends="cryptsetup-devel"
+short_desc="Library for storing metadata in the LUKSv1 header"
+maintainer="Johannes Heimansberg <git@jhe.dedyn.io>"
+license="LGPL-2.0-or-later"
+homepage="https://github.com/latchset/luksmeta"
+distfiles="https://github.com/latchset/luksmeta/archive/v$version/luksmeta-$version.tar.gz"
+checksum=0eea7d50a0411e0c1e383fd47073806ed7d435b27410504e33bfbc792a1688fc
+make_check="no" # fails on some architectures
```

Comment:
fails how? on which architectures?

Reply:
One of the tests failing tries to create a LUKS container in a 4 MB file and doesn't succeed doing so because apparantly that's too small, again on x86_64-glibc. I don't know why it uses such a small size.
As for the other tests, I don't know what exactly they are trying to test.

```
Device /tmp/luksmeta.3yxzFZhvt7 is too small for activation, there is no remaining space for data.
```

Reply:
would be good to ask upstream about this

Reply:
I'll do that.

Reply:
I've opened a [bug report](https://github.com/latchset/luksmeta/issues/15) with upstream.